Geometry/SphereSurfaceArea.js

```diff
@@ -0,0 +1,7 @@
+// https://en.wikipedia.org/wiki/Sphere rounds to 2 decimal places
```

Comment:
Use a proper JSDoc comment and don't arbitrarily round to two decimal places. You may round to two decimal places in the tests though.